### ISSUE TITLE
add alt text for nbImage

### ISSUE THREAD

**Issue by pietroppeter** [pietroppeter/nimib#193]
currently the nbImage block has only two parameters: `url` and `caption` and `caption` is used for alt text both in html and markdown backend. We should add an alt text parameter to be used correctly since caption and alt text in general should be different. We could still keep the old behaviour of using caption when alt text is missing. raised when reviewing #192
